studygroups/management/commands/rename_studygroups.py

```diff
@@ -0,0 +1,18 @@
+from django.core.management.base import BaseCommand, CommandError
```

Comment:
@s-kennedy this should be a data migration rather than a custom command. I didn't try to run it, but looking at the code I think ```course = Course.objects.filter(pk=int(sg.course.id)).first()``` is unnecessary - `sg.course.id` already implies an existing course associated with the StudyGroup and the if condition doesn't skip the code below it, so it's effectively a no-op